### PR TITLE
Core update from 9.0 to 9.1

### PR DESCRIPTION
**Pull request by ben-hamelin**
## Summary

Update profile to move to core 9.1.
Removes/updates patches as needed.
Remove commented PHP version from github actions workflow.
Updates QueueWorker class to use new exception class (related to patch from issue 1832818 - Allow a queue item to be postponed, which we have removed).
Moves cache_flush in the ecms_workflow module from the __desctruct method to the .module. This was causing an error during site installs.
Update views_ajax_get to 1.x-dev to get latest bug fixes/updates.
Restores functional tests for Github actions, they are completing now.

Once this is merged and tagged, we will need to update the complementary ecms_distribution PR: https://github.com/State-of-Rhode-Island-eCMS/ecms_distribution/pull/101
## Metadata

| Question | Answer |
|----------|--------|
| Did you use a meaningful pull request title? | yes
| Did you apply meaningful labels to the pull request? | yes
| Did you perform a self review first? | yes
| Documentation reflects changes? | no
| `CHANGELOG` reflects changes? | yes
| Unit/Functional tests reflect changes? | yes (minor updates)
| Did you perform browser testing? | yes
| Risk level |  High
| Relevant links | https://thinkoomph.jira.com/browse/RIGA-24

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -64,17 +64,12 @@
...
-        "2914389 - Allow profiles to exclude dependencies of their parent": "https://www.drupal.org/files/issues/2018-07-09/2914389-8-do-not-test.patch",
-        "3106659 - Error: Call to a member function getItemDefinition() on null in media_requirements()": "https://www.drupal.org/files/issues/2020-04-29/media-install-requirements-3106659.patch",
+        "1356276 - Allow profiles to define a base/parent profile and load them in the correct order": "https://www.drupal.org/files/issues/2020-12-13/1356276-633-by-mpotter-balsama-phenaproxima-9.1.0-9.2.x.patch",
         "2794431 - [META] Formalize translations support (#102)": "https://www.drupal.org/files/issues/2020-09-29/jsonapi_add_and_update_translations_support-2794431-102.patch",
-        "3083051 - Remove tabledrag.js replacement when core issues are resolved": "https://www.drupal.org/files/issues/2020-10-02/3083051-21.patch",
-        "1832818 - Allow a queue item to be postponed": "https://www.drupal.org/files/issues/2020-03-24/1832818.80-requeue.patch",
```
Comment (pfrilling) [inline composer.json:72]: This was a requirement for the syndication piece. Flagging as it might need some manual testing.

**Review by pfrilling — APPROVED** [review]
I think this all looks good. Has any manual testing happened with the `DelayedRequeueException`? I just want to ensure it continues processing the other items in the queue after it delays one.

**Comment by pfrilling** [comment]
@ben-hamelin, Should we re-enable the existing site testing too and let it run once?

**Comment by ben-hamelin** [comment]
> @ben-hamelin, Should we re-enable the existing site testing too and let it run once?

@pfrilling I tried that on commit `00ed6e9` but they threw errors.

File: composer.json (hunk 1 above)
Comment (ben-hamelin) [reply composer.json:72]: What's the best way to test this? Create a few new sites, register them, and then delete one? Then queue up a notification to make sure all the others get through?

**Pull request merged**